Algorithm/Examples/BubbleAlgorithm.cs

```diff
@@ -46,7 +46,9 @@ public class BubbleAlgorithm : QCAlgorithm
         Dictionary<string, RelativeStrengthIndex> rsiDic = new Dictionary<string, RelativeStrengthIndex>();
         Dictionary<string, MovingAverageConvergenceDivergence> macdDic = new Dictionary<string, MovingAverageConvergenceDivergence>();
 
-
+		/// <summary>
+		/// Initialise the Algorithm and Prepare Required Data.
```

Comment:
it looks like there's some tab spacing issues here. Sadly this isn't documented, but by convention we use spaces. It may be an easy fix, find/replace the tabs with spaces. We use 1 tab=4 spaces. Also, it looks like the indentation doesn't line up with the method definition.